tests/Microsoft.Bot.Builder.TestBot.Json/Samples/RandomGenerator/RandomGenerator.main.dialog

```diff
@@ -0,0 +1,37 @@
+﻿{
+    "$schema": "../../testbot.schema",
+    "$kind": "Microsoft.AdaptiveDialog",
+    "triggers": [
+        {
+            "$kind": "Microsoft.OnBeginDialog",
+          "actions": [
+            {
+              "$kind": "Microsoft.EmitEvent",
+              "eventName": "setTestOptions",
+              "EventValue": {
+                "randomSeed": 100
+              }
```

Comment:
This can be an arbitrary object put into conversation.testOptions.  You don't need to strongly type this.